mmpose/models/backbones/mobilenet_v3.py

```diff
@@ -86,8 +86,6 @@ def __init__(self,
             raise ValueError('frozen_stages must be in range(-1, '
                              f'{len(self.arch_settings[arch])}). '
                              f'But received {frozen_stages}')
-        self.out_indices = out_indices
-        self.frozen_stages = frozen_stages
```

Comment:
why these two lines are deleted?

Reply:
They are reduplicate. Line 94 & 95 are the same in the original code .